rotkehlchen/chain/ethereum/modules/compound/v2/compound.py

```diff
@@ -103,21 +115,168 @@ def _get_apy(self, address: ChecksumEvmAddress, supply: bool) -> FVal | None:
         apy = ((FVal(rate) / ETH_MANTISSA * BLOCKS_PER_DAY) + 1) ** (DAYS_PER_YEAR - 1) - 1
         return apy
 
+    def _get_v3_interest_rates(
```

Comment:
So I have an issue with placements here.

1. This file is becoming huge and functions pretty big.
2. This is in `modules/v2/compound.py`. While the functions you are introducing are about v3. So they are not where they should be and the code organization breaks down.